shuup/admin/static_src/base/js/main-menu.js

```diff
@@ -31,11 +31,19 @@ const handleMainMenu = () => {
     submenu.classList.add('active');
     categoryArrow.classList.add('rotate');
   };
-
+  const toggleBtn = document.getElementById('menu-button');
```

Comment:
could you explain why these are needed for since the new menu "toggle-button" shouldn't add any new functionality but just do exact same as with the current "X-button" at top-navigation.